bin/t2-identify.js

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env node
+"use strict";
+
+let got = require('got');
+let checkForGypFile = require('../lib/check-for-gyp');
+
+function logPackageInfo(doc){
+  let name = doc.name;
+  let version = doc["dist-tags"] && doc["dist-tags"].latest;
+  console.log('binding.gyp detected', name + "@" + version);
+}
+
+function logReject(error){
```

Comment:
Maybe move logging into another file? Similar to https://github.com/tessel/t2-cli/blob/master/lib/logs.js